Streetcode/Streetcode.XIntegrationTest/ControllerTests/Utils/Client/Base/BaseClient.cs

```diff
@@ -5,23 +5,23 @@ namespace Streetcode.XIntegrationTest.ControllerTests.Utils.Client.Base
 {
     public class BaseClient
     {
-        private readonly RestClient _client;
-
-        protected readonly string _secondPartUrl;
+        private readonly RestClient client;
 
         public BaseClient(HttpClient client, string secondPartUrl = "")
         {
-            _client = new RestClient(client) { AcceptedContentTypes = ContentType.JsonAccept };
-            _secondPartUrl = secondPartUrl;
+            this.client = new RestClient(client) { AcceptedContentTypes = ContentType.JsonAccept };
+            this.SecondPartUrl = secondPartUrl;
         }
 
+        protected string SecondPartUrl { get; }

```

Comment:
Better to write this property before constructor (under `private readonly RestClient client;` prop).

Reply:
[SA1201](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation%2FSA1201.md)